Makefile

```diff
@@ -165,10 +165,15 @@ test-generic:
 test-cmd-login-logout:
 	ginkgo $(GINKGO_FLAGS_SERIAL) -focus="odo login and logout command tests" tests/integration/loginlogout/
 
-# Run link and unlink command tests
-.PHONY: test-cmd-link-unlink
-test-cmd-link-unlink:
-	ginkgo $(GINKGO_FLAGS) -focus="odo link and unlink command tests" tests/integration/
+# Run link and unlink commnad tests against 4.x cluster
+.PHONY: test-cmd-link-unlink-4-cluster
+test-cmd-link-unlink-4-cluster:
```

Comment:
shouldn't we add this in the openshift CI test run as well?

Reply:
Link unlink is already under `make test-integration`  target, so I guess we don't need to add.

Reply:
but didn't you just rename it?